### DISCUSSION TITLE
Use ANSI color codes in script

### DISCUSSION THREAD

**Discussion by og900aero** [polybar/polybar#3185] [Q&A]
Hello. I use polybar 3.7.2 and custom script module for volume change and icon (other icon when volume on and other icon when volume off).
I use my script this line:
`echo -e "\033[1;48;5;128m\033[0m $curVol%"`

But when restart polybar, I get error message and wrong volume display:

![Untitled](https://github.com/user-attachments/assets/1fe11e0e-1567-4ad4-8376-a70a6636fb4e)

I use this web page for help: https://misc.flogisoft.com/bash/tip_colors_and_formatting

**Comment by patrick96** [comment]
Polybar doesn't use or support ANSI escape codes. It uses its own [formatting tags](https://github.com/polybar/polybar/wiki/Formatting#format-tags)
